### PR TITLE
Configure applications based on load tests

### PR DESCRIPTION
**Pull request by alvarolop**

### CODE REVIEW

**Comment by ernesgonzalez33** [comment]
The E2E test is not passing. Do we know why?

File: fiware-platform/values.yaml [hunk 1]
```diff
@@ -80,7 +80,7 @@ applications:
     - values.yaml
 
   - name: airquality-simulator
-    enabled: true
+    enabled: false
```
Comment (ernesgonzalez33) [inline fiware-platform/values.yaml:83]: Just a reminder to turn this to true again before merging into main

Comment (alvarolop) [reply fiware-platform/values.yaml:83]: Good point! We forgot to tell you, setting it to false does not disable it

File: documentation/LOAD_TESTING.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Load Testing
+
+You can run load tests to calculate the maximum throughput for your deployment. This section shows how to run tests using the FIWARE load testing tool and some reports of tests run in a demo scenario.
+
+## 1. Run your own load tests
+
+1. Clone [load-tests](https://github.com/FIWARE/load-tests) repository locally.
+2. Configure the load test on [/src/test/resources/test.conf]()
```
Comment (ernesgonzalez33) [inline documentation/LOAD_TESTING.md:8]: This link is sending us to our repo docs. I think you want to go here, right?

https://github.com/FIWARE/load-tests/blob/master/src/test/resources/test.conf

Or maybe you are just pointing to the local directory after the cloning, which then we need to remove the link

**Review by jtudelag — APPROVED** [review]
LGTM!

**Pull request merged**